koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -97,13 +98,18 @@ def create_daily_archives(
             date_range = {"start": min(days), "end": max(days), "invoice_month": str(invoice_month)}
             partition_dates = invoice_month_data.partition_date.unique()
             for partition_date in partition_dates:
+                if partition_date_counter.get(partition_date) is None:
```

Comment:
Unless we need an explicit check for `None`, a truthiness check is more idiomatic.
```suggestion
                if not partition_date_counter.get(partition_date):
```

Reply:
It was a choice since its a numeric counter where 0 could be a value and not necessarily intended to be considered false technically resetting it to 0 would be fine, but the expectation is that the absence here would be an explicit None.

Reply:
That makes perfect sense.